src/essential/Essential.jl

```diff
@@ -40,10 +46,10 @@ export  @model,
         ADBackend,
         setadbackend,
         setadsafe,
-        ForwardDiffAD,
-        TrackerAD,
-        ZygoteAD,
-        ReverseDiffAD,
+        AutoForwardDiff,
+        AutoTracker,
+        AutoZygote,
+        AutoReverseDiff,
```

Comment:
Do we actually want to export these types? Or should we tell users to use `ADTypes.AutoForwardDiff` etc. (in particular when these types would be used in other packages such as e.g. AdvancedVI as well).